web/components/input/buttons/PrimaryButton/PrimaryButton.sass

```diff
@@ -4,7 +4,6 @@ button.primary
   color: $white
 
   border-radius: 4px
-  min-width: 80px
```

Comment:
why are we removing `min-width` here?

Reply:
this sets to min-width to text + padding rather than a flat `80px` which causes the wonky clipping on smaller screens

Reply:
interesting, good to know!